src/BIDS-Validator-info.md

```diff
@@ -0,0 +1,158 @@
+
+The [BIDS Validator](https://github.com/bids-standard/bids-validator) is a tool that checks if a dataset is compliant with the BIDS standard. The validator is available for use within several different environments to best suit individual user preferences and use cases, those versions are:
+
+- A web browser based version
+- Command line version
+- Docker based version
+- A python library installable via pip
+
+Instructions to install and use these versions can be found within the [quickstart guide](https://github.com/bids-standard/bids-validator#quickstart) at the BIDS Validator repository.
+
+**Data Privacy and Confidentiality** 
+
+Please note that the web app is entirely browser(not server) based. As such, there is no file uploading as part of the validation.
+
+# How to use
+
+## Browser Version
+
+1) The BIDS Validator can be found at http://bids-standard.github.io/bids-validator/. It requires that you use the Chrome or Firefox browser, since those are the only ones in which you can select a whole folder rather than individual files. 
+
+![home](https://i.imgur.com/YD38eTE.png)
```

Comment:
I did not realize but we will have to also get those images and store them in the repo.